labonneboite/web/api/views.py

```diff
@@ -439,7 +459,10 @@ def create_hidden_market_fetcher(location, request_args):
     if request.args['user'] in settings.API_INTERNAL_CONSUMERS:
         kwargs['flag_pmsmp'] = check_bool_argument(request_args, 'flag_pmsmp', 0)
 
-    return search.HiddenMarketFetcher(location, **kwargs)
+    if location is not None:
```

Comment:
j'ai un doute mais je crois bien que `if location:` serait équivalent et plus lisible. A tester.

Reply:
visiblement la bonne pratique est de garder is not None, si je comprends bien [cette doc officielle](https://www.python.org/dev/peps/pep-0008/) qui dit 

> Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

Ok pour toi?

Reply:
Yeah ok! `if x is not None` est plus explicite et safe. Lu aussi sur https://amir.rachum.com/blog/2012/08/25/you-cant-handle-the-truth/